packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -8,6 +8,15 @@ import * as Selectors from '../helpers/selectors';
 
 const { expect } = chai;
 
+// mongodb-runner defaults to stable if the env var isn't there
+const MONGODB_VERSION = (process.env.MONGODB_VERSION || '5.0.6')
```

Comment:
I'm trying to go with the version number that we asked for and therefore think we should be running as opposed to the version we are running that we could probably have pulled from the connection to the server.

There are arguments either way, but what I like about this one is that it is just a little more brittle. Enough so that it also tests our test/setup/CI assumptions.

My thinking is that if it becomes a pain we can always change it.

The main problem with just pulling the version number from the connection is that if we never specify a specific version and we just get something arbitrary then the tests would always pass because we always get the same version for every task and it might take a while for us to realise that we broke something.